package.json

```diff
@@ -2,33 +2,36 @@
   "name": "react-adaptive-hooks",
   "version": "0.0.5",
   "description": "Give users a great experience best suited to their device and network constraints",
-  "main": "index.js",
   "repository": "https://github.com/GoogleChromeLabs/react-adaptive-hooks",
   "author": "Google Chrome",
   "license": "Apache-2.0",
   "private": false,
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "unpkg": "dist/index.umd.js",
+  "source": "index.js",
   "scripts": {
+    "build": "microbundle",
+    "dev": "microbundle watch",
     "test": "jest"
   },
   "peerDependencies": {
     "react": "^16.8.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.5.5",
-    "@babel/node": "^7.5.5",
     "@babel/preset-env": "^7.5.5",
-    "react": "16.9.0",
-    "react-dom": "16.9.0",
-    "ua-parser-js": "^0.7.20",
     "@testing-library/react-hooks": "^1.1.0",
-    "babel-polyfill": "^6.26.0",
     "jest": "^24.8.0",
-    "react-test-renderer": "16.9.0"
+    "microbundle": "0.11.0",
+    "react": "16.9.0",
+    "react-dom": "16.9.0",
+    "react-test-renderer": "16.9.0",
+    "ua-parser-js": "^0.7.20"
```

Comment:
Minor: I believe it should be safe to drop `ua-parser-js` too (happy for us to do this separately too).